tensorflow_addons/examples/layers_weightnormalization.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>Can we align them?</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Done. Thanks.